src/lib/chips/chip.ts

```diff
@@ -231,18 +239,23 @@ export class MatChip extends _MatChipMixinBase implements FocusableOption, OnDes
         terminateOnPointerUp: globalOptions.terminateOnPointerUp,
       };
     }
+
+    if (this._animationsDisabled) {
```

Comment:
This should not be necessary since these animations should automatically run in `0` time a `NoopAnimationsModule` environment correct?

Reply:
The `RippleRenderer` doesn't have access to the `ANIMATION_MODULE_TYPE` so we have to disable it from here.